### ISSUE TITLE
Update design details of Ordinals inscription send flow

### ISSUE THREAD

**Issue by mica000** [blockstack/blockstack-app#3350]
To be consistent with BTC experience UI needs to be adapted to the fullpage when on a popover

<img width="420" alt="Screenshot 2023-03-03 at 14 01 34" src="https://user-images.githubusercontent.com/119523541/222727676-69665bd5-567f-4e21-9249-def64652640c.png">

Designs: https://www.figma.com/file/OSQocEEJnfcMkniEl8u3U8/%231483---Initial-Bitcoin-support?node-id=1540%3A73103&t=FNRziBxZd0VBTFeU-4
<img width="371" alt="Screenshot 2023-03-03 at 14 03 43" src="https://user-images.githubusercontent.com/119523541/222728019-431ab3e1-afc2-4af2-a5e4-e431b480b880.png">

**Comment by mica000** [comment]
We also need to update the copy on all the steps
<img width="1688" alt="2" src="https://user-images.githubusercontent.com/119523541/222728700-76831f89-8efa-4030-8773-f97bbb14d420.png">
<img width="1688" alt="3" src="https://user-images.githubusercontent.com/119523541/222728703-b38f9673-afbb-49f6-b84b-584533e5e8fa.png">
<img width="1688" alt="4" src="https://user-images.githubusercontent.com/119523541/222728706-323db02a-a258-418e-b4b1-04ddfcb4eec5.png">

**Comment by markmhendrickson** [comment]
@mica000 Are the fixes in this issue still relevant?

**Comment by mica000** [comment]
@markmhendrickson yes, will be solved in https://github.com/leather-wallet/extension/issues/4371

**Comment by markmhendrickson** [comment]
It seems there are more fixes in this particular issue than #4371 per se – in particular the copy changes from [your earlier comments](https://github.com/leather-wallet/extension/issues/3350#issuecomment-1453514962)?

**Comment by markmhendrickson** [comment]
I'm noticing some differences between the Figma and implementation during usage day:

- Top box has black background in prod vs. gray in Figma
- The "Ordinals inscription" unit below is joined with the address field visually in Figma vs. not in prod
- The address field doesn't have a "To" label in production as seen in Figma
- We say "Inscription 2652322" up top in prod but it's "#2652322" in Figma
- We have both back and close buttons in the modal nav in prod but only back in Figma
- There's a "View details" link in Figma but not in prod

@mica000 are these all worth tackling still?

<img width="400" alt="Screenshot 2023-11-21 at 15 54 43" src="https://github.com/leather-wallet/extension/assets/28991/e0ffac92-24e9-4e82-aa63-04eb02284e46">

**Comment by markmhendrickson** [comment]
@mica000 assigning you since these details will be resolved with standardization work in progress
